app/models/casa_admin.rb

```diff
@@ -53,5 +53,5 @@ def deactivate
 #
 # Foreign Keys
 #
-#  fk_rails_...  (casa_org_id => casa_org.id)
+#  fk_rails_...  (casa_org_id => casa_orgs.id)
```

Comment:
this is an unwanted change, how did this get here?

Reply:
I'm not sure, but I did it last week according to a git tracking extension I got a few days ago.  Its way off of any code I was working with.  Best guess is I clicked on the page at random, maybe while googling something, and hit backspace or delete.